manuscript/Manuscript.qmd

```diff
@@ -1781,20 +1781,20 @@ This normalisation ensures that indicators are more comparable because an indica
 Similarly, a value of 0 means that the variable is in the worst possible state, for example that the species or ecosystem function in question is completely lost from the ecosystem.
 The reference condition needs to be defined for each ecosystem condition assessment separately, but SEEA EA gives some suggestions, such as an an ecosystem with no or minimal anthropogenic disturbance.
 
-The SEEA EA framework recommends that ecosystem condition indicators give an unbiased representation of the condition for a given area [table 1 in @czucz_selection_2021; see also @unitednations2021 §2.87], and that condition indicators are recorded uniquely for each ecosystem asset (EA) so that the condition in the ecosystem accounting area (EAA) can be inferred from an area weighted mean based on the relative extents of the EAs [@unitednations2021 §5.54].
-Ecosystem assets are defined as "ecological entities" (meaning areas) about which information is sought and about which statistics are compiled [@unitednations2021].
+The SEEA EA framework recommends that ecosystem condition indicators give an unbiased representation of the condition for a given area [table 1 in @czucz_selection_2021; see also @unitednations2021 §2.87], and that condition indicators are recorded uniquely for each ecosystem asset (EA) so that the condition in the ecosystem accounting area (EAA) can be inferred from an area weighted mean based on the relative extents of the ecosystem assets [@unitednations2021 §5.54].
+Ecosystem assets (EAs) are defined as "ecological entities" (meaning areas) about which information is sought and about which statistics are compiled [@unitednations2021].
```

Comment:
EA was explained in the previous sentence, but I will see if I even need this abbreviation at all.

Reply:
I will get rid of this abbreviation. Even though I used it quite often, it is the same abbreviation commonly used for ecosysyem accounts (e.g. in SEEA EA)